docs/installation.md

```diff
@@ -57,3 +71,12 @@ root user:
 3. Make sure your desktop users are part of the `plugdev` group, by running
    `gpasswd -a <desktop username> plugdev`. If these users were not assigned to the
    group before, they must re-login for the changes to take effect.
+
+
+Then solaar can be run from the download directory without using sudo.
+
+Solaar can be installed as a user python program from the solaar directory using `pip install --user .`
+and then run as `~/.local/bin/solaar`.
+Installing python programs to system directories using pip is somewhat frowned on but
+installing to /usr/local can be done via `sudo bash -c 'umask 022 ; pip install .'`
+in the solaar directory.  Then solaar can be run as /usr/local/bin/solaar.
```

Comment:
I am a bit of a noob with using Python. Could you add some info about what the command do and why they are recessary.

Reply:
I'm not sure what could be added here.   pip is the python installer.  It is a bit of a Swiss-army-knife.  In this case it is running setup.py to do the work, as it is doing a local install.
This is analogous to saying run `make install` or `make install --prefix=~` to install a system written in C or C++.

Reply:
I think we write these installation instructions for the more noob users. So tell what pip is and why you need to run the command `pip install --user .`. Try to see it from the noobs perspective.
If we are going to need pip, it should be part of the requirements.
All your other changes are great. Thank you for your pull request.

Reply:
OK, I added some more on pip.  I don't want to go too far into the weeds though.
pip should be available on any system that has python.
pip isn't necessary to run solaar - as said in the document, solaar can be run directly from the download directory.

Reply:
@pfps: Thank you for the extra effort. Looks great! :-)